tgui/packages/tgui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "private": true,
+  "name": "tgui",
+  "version": "3.1.0",
+  "dependencies": {
```

Comment:
Most of these are `devDependencies` rather than `dependencies`. Won't make a difference in how we use it, but good to be correct I think?

Reply:
it's much easier to work with plain dependencies, because you don't need to pass extra flags to yarn/npm when adding/upgrading packages.